RecoHI/HiJetAlgos/plugins/UETableProducer.cc

```diff
@@ -0,0 +1,232 @@
+// system include files
```

Comment:
@slava77 
I am going to ask @yslai to move this kinf of condition producers into `CondTools`.
We can agree on a policy: hold a PR in case you find DB producers.
Again, sorry for being slow...